tests/pr_test.go

```diff
@@ -2,30 +2,55 @@
 package test
 
 import (
+	"log"
 	"os"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/cloudinfo"
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
+	"gopkg.in/yaml.v3"
 )
 
 const resourceGroup = "geretain-test-base-ocp-vpc"
 const standardExampleTerraformDir = "examples/standard"
 
+// Define a struct with fields that match the structure of the YAML data
+const yamlLocation = "../common-dev-assets/common-go-assets/common-permanent-resources.yaml"
+
+type Config struct {
+	ExistingAccessTags []string `yaml:"accessTags"`
+}
+
 // Ensure there is one test per supported OCP version
 const ocpVersion1 = "4.12"
 const ocpVersion2 = "4.11"
 const ocpVersion3 = "4.10"
 const ocpVersion4 = "4.9"
 
 var sharedInfoSvc *cloudinfo.CloudInfoService
+var existingAccessTags []string
 
 // TestMain will be run before any parallel tests, used to set up a shared InfoService object to track region usage
 // for multiple tests
 func TestMain(m *testing.M) {
-
 	sharedInfoSvc, _ = cloudinfo.NewCloudInfoServiceFromEnv("TF_VAR_ibmcloud_api_key", cloudinfo.CloudInfoServiceOptions{})
+
+	// Read the YAML file contents
```

Comment:
There is now an inbuilt function to do this called `LoadMapFromYaml` (see https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/blob/main/tests/pr_test.go#L141). It makes the code easier if you wanted to roll it out here?